gamemodes/terrortown/gamemode/shared/lang/fr.lua

```diff
@@ -448,47 +448,47 @@ Teleporte vers un lieu marqué.
 La téléportation fait du bruit, et le nombre d'utilisations est limité.]]
 
 -- Ammo names, shown when picked up
-L.ammo_pistol = "Munitions 9mm"
+L.ammo_pistol = "Munitions de 9mm"
 
-L.ammo_smg1 = "Munitions SMG"
-L.ammo_buckshot = "Munitions de fusil à pompe"
-L.ammo_357 = "Munitions fusil"
-L.ammo_alyxgun = "Munitions Deagle"
-L.ammo_ar2altfire = "Munitions de pistolet de détresse"
-L.ammo_gravity = "Munitions de Poltergeist"
+L.ammo_smg1 = "Munitions de SMG"
+L.ammo_buckshot = "Munitions de fusil À pompe"
+L.ammo_357 = "Munitions de fusil"
+L.ammo_alyxgun = "Munitions de Deagle"
+L.ammo_ar2altfire = "Munitions du pistolet de détresse"
+L.ammo_gravity = "Munitions du Poltergeist"
 
 -- Round statut
 L.round_wait = "En attente"
-L.round_prep = "Préparation"
+L.round_prep = "PrÉparation"
```

Comment:
This looks like a mistake

Reply:
i have use É instead of  é  its because of this 
![Capture d’écran 2021-01-06 103620](https://user-images.githubusercontent.com/73431903/103753413-4eb0bd00-500b-11eb-989f-2b5ee6f23c87.png)
![Capture d’écran 2021-01-06 103620](https://user-images.githubusercontent.com/73431903/103753415-4f495380-500b-11eb-95d7-3833a7e1451d.png)

Reply:
and that the same issue for every É Ô À on the F1 menu

Reply:
![Capture d’écran 2021-01-06 113627](https://user-images.githubusercontent.com/73431903/103759155-91769300-5013-11eb-9c6c-99805da42269.png)
 on the F1 menu thats look like this